draft-ietf-httpbis-resumable-upload.md

```diff
@@ -88,7 +88,7 @@ In this example, the client first attempts to upload a file with a known size in
 
 An informational response can be sent to the client to signal the support of resumable upload on the server and transmit the upload resource URL in the Location header.
 
-~~~
+~~~ aasvg
```

Comment:
If you do this, you should also add `package.json` to the repository.  You might also need to add `package-lock.json` to the `.gitignore` file.  Otherwise, people without aasvg installed will have a hard time.

That's a once-only thing that you have the privilege of having to do.